Source/WebKit/WebProcess/WebPage/ios/WebPageIOS.mm

```diff
@@ -163,11 +165,20 @@
 #import <WebCore/PromisedAttachmentInfo.h>
 #endif
 
+#if USE(APPLE_INTERNAL_SDK)
+#import <AXRuntime/AXDefines.h>
+#import <AXRuntime/AXNotificationConstants.h>
+#endif
+
 #import <pal/cocoa/RevealSoftLink.h>
 
 #define WEBPAGE_RELEASE_LOG(channel, fmt, ...) RELEASE_LOG(channel, "%p - WebPage::" fmt, this, ##__VA_ARGS__)
 #define WEBPAGE_RELEASE_LOG_ERROR(channel, fmt, ...) RELEASE_LOG_ERROR(channel, "%p - WebPage::" fmt, this, ##__VA_ARGS__)
 
+#if !USE(APPLE_INTERNAL_SDK)
+#define kAXPidStatusChangedNotification 0
+#endif
```

Comment:
Our modern practice is to put these into `SPI.h` headers, not just into the one file where they are used. Sprinkling them into different files leads to a mess over time, and we would like to avoid that.